### ISSUE TITLE
Make translations easier by less text fragmentation

### ISSUE THREAD

**Issue by JoKeyser** [jasp-stats/jaspPower#15]
First of all, thank you for adding power analyses to JASP, this is super exciting!

During my first translations into German, I notice that some strings are relatively fragmented, and then put back together in code (example below).
I understand that this may make sense to reduce code and typing for the English language, but unfortunately, it makes translation harder, and probably even impossible at times:

- Translators often cannot know what strings are put together, unless they read and understand the code. On Weblate (and probably other tools), it's not possible to clarify this kind of context.
- The grammar rules of languages are too different to generally make this kind of text manipulation possible in code.
  It's better to write out the entire sentence, because any word or concept may end up in a totally different position; but in the code, it's always pasted into a fixed position.

Here is one example, from file [pwr_ttestis.R#L831](https://github.com/jasp-stats/jaspPower/blob/db6b91c5d8ba3bd9ad8003012eba43aaa44bb43f/R/pwr_ttestis.R#L831).
Note how the string is fragmented into two pieces.
I hope it becomes clearer what I mean?

```R
n_text <- ifelse(n1 == n2,
    gettextf("a sample size of %1$s in each group", n1),
    gettextf("group sample sizes of %1$s and %2$s, respectively", n1, n2)
  )

# ... some other code ...

gettextf(
   "Both assume %1$s.",
    n_text
),
```

Again, thank you for the great work.

**Comment by vandenman** [comment]
Actually, we already have this in the guide [here](https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/jasp-human-guide.md#writing-for-internationalization). But we have multiple guides, and there is a lot of material in there. We should probably review them and see how we can make things easier for new (and old) contributors.

@JoKeyser please feel free to create issues for this whenever you encounter them. In my opinion, untranslatable strings are basically bugs.

**Comment by JoKeyser** [comment]
Ok thank you both for understanding and this general issue and for taking translation so seriously.

Not sure about the scope of this issue; pointing out every single occurrence of fragmented strings may be too tedious? (But it could be done...) For this module, maybe this one is sufficient for now?

And regarding other modules, it's probably best to open an issue in the general issues and scoping it for the module, so the module maintainers can be assigned and deal with it?

**Comment by vandenman** [comment]
> For this module, maybe this one is sufficient for now?

Yes, I'll go over every `gettext` and `gettextf` and make changes as I deem fit.

>  it's probably best to open an issue in the general issues and scoping it for the module, so the module maintainers can be assigned and deal with it?

Yes, one issue per module should be fine.
